pkg/tarsum/fileinfosums.go

```diff
@@ -2,7 +2,8 @@ package tarsum
 
 import "sort"
 
-// This info will be accessed through interface so the actual name and sum cannot be medled with
+// FileInfoSumInterface This info will be accessed through interface
+// so the actual name and sum cannot be medled with
```

Comment:
"FileInfoSumInterface provides an interface for accessing file checksum information within a tar file. This info is accessed through interface so the actual name and sum cannot be medled with."